python/semantic_kernel/agents/azure_ai/azure_ai_agent.py

```diff
@@ -245,7 +321,7 @@ async def invoke_stream(
         parallel_tool_calls: bool | None = None,
         metadata: dict[str, str] | None = None,
         **kwargs: Any,
-    ) -> AsyncIterable["ChatMessageContent"]:
+    ) -> AsyncIterable[ChatMessageContent]:
```

Comment:
Should we consolidate all `invoke_stream` return types to `StreamingChatMessageContent` since the Agent's ABC now specifies that?